clocks/clock.go

```diff
@@ -1,56 +1,14 @@
 package clocks
 
 import (
-	"context"
-	"time"
+	clocks "github.com/vimeo/go-clocks"
```

Comment:
nit: we can remove the `clocks` alias here

Reply:
Thanks for pointing out that this doesn't match.

Since the package name doesn't match the last component of the package-path, I like to make the package name explicit in the import statement.
I've updated all the imports of `go-clocks` and `go-retry` to provide explicit package names.